### PR TITLE
fix example for PodTolerationRestriction annotation

### PR DESCRIPTION
**Pull request by danielkucera**
defaultTolerations has to be unmarshall-able as an array

### CODE REVIEW

File: content/en/docs/reference/access-authn-authz/admission-controllers.md [hunk 1]
```diff
@@ -706,7 +706,7 @@ kind: Namespace
 metadata:
   name: apps-that-need-nodes-exclusively
   annotations:
-    scheduler.alpha.kubernetes.io/defaultTolerations: '{"operator": "Exists", "effect": "NoSchedule", "key": "dedicated-node"}'
+    scheduler.alpha.kubernetes.io/defaultTolerations: '[{"operator": "Exists", "effect": "NoSchedule", "key": "dedicated-node"}]'
     scheduler.alpha.kubernetes.io/tolerationsWhitelist: '{"operator": "Exists", "effect": "NoSchedule", "key": "dedicated-node"}'
```
Comment (tengqm) [inline content/en/docs/reference/access-authn-authz/admission-controllers.md:710]: should we change line 710 as well?

Comment (danielkucera) [reply content/en/docs/reference/access-authn-authz/admission-controllers.md]: correct, just tested that and fixed in latest commit

**Pull request merged**